.github/workflows/ci_tests.yaml

```
@@ -87,7 +87,7 @@ jobs:
     steps:
       # Checkout current git repository
       - name: Checkout
-        uses: actions/checkout@v4.0.0
+        uses: actions/checkout@v4.1.0
         with:
           # fetch all history so that setuptools-scm works
           fetch-depth: 0
```

Try the treeless clone in `actions/checkout@v4.1.0` to see if it speeds up `git checkout` (currently about 15s on Windows)? Xref https://github.com/actions/checkout/pull/1396 and https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone.

```suggestion
        uses: actions/checkout@v4.1.0
        with:
          # fetch all history with treeless clone so that setuptools-scm works
          filter: tree:0
```

---

it seems the pygmt version string is still incorrect.

---

Hmm yeah, it's only a few seconds faster too, so not really worth it perhaps. Ok to revert the change and merge.